src/lora_plan.erl

```diff
@@ -604,10 +610,10 @@ dualplan_region(GatewayRegion, Freq, _DataRate) ->
                 true -> 'AU915_SB5';
                 false -> GatewayRegion
             end;
-        'AS923_1B' ->
-            IsAU915DP = find_frequency(Freq, DP_List),
-            case IsAU915DP of
-                true -> 'AU915_DP';
+        'AS923_1C' ->
+            IsAU915_1C = find_frequency(Freq, SB5_List),
```

Comment:
I can't tell if something in here is a typo.

`AS923_1` clause is checking for Subband 2 in `SB5_List` and returning `AU915_SB5`.
`AS293_1C` clause is checking for AU 1C in `SB5_List` and also returning `AU915_SB5`.

Of which region is `SB5_List` referring to?

Reply:
Both should be the exact same logic.

They are both checking for any of the 6 frequencies of 8 in sub-band 5.  They exclude the 2 Join frequencies from AS923.
If any of those frequencies are found the SB5 region is returned to the caller.